cmd/apps/therealssh/therealssh.go

```diff
@@ -32,6 +34,9 @@ func main() {
 	}
 
 	ssh.Debug = *debug
+	if !ssh.Debug {
+		logging.SetLevel(logrus.InfoLevel)
```

Comment:
Would you consider getting rid of a `logrus` dependency here using `logging.LevelFromString`? This way abstracts the logger implementation from this package. I'd write the following:
```suggestion
		if infoLevel, err := logging.LevelFromString("info"); err == nil {
			logging.SetLevel(infoLevel)
		}
```

Reply:
@nkryuchkov I personally think this is not necessary. `logging` depends on `logrus` anyway so we won't we having unnecessary dependencies.